src/index.ejs

```diff
@@ -13,6 +13,7 @@
    <script type="text/javascript"
      src="https://maps.googleapis.com/maps/api/js?libraries=visualization&key=AIzaSyB-U6Gw1USZvOGJWTlD9yjrM46HpaZu-3Q">
    </script>
+   <script src="//js.pusher.com/4.0/pusher.min.js"></script>
```

Comment:
You don't need to install `pusher` via CDN, because you already installed via npm